src/__tests__/02c-onchain-send.ts

```diff
@@ -184,12 +188,22 @@ it("makes onchain on-us transaction with memo", async () => {
   })
   expect(paymentResult).toBe(true)
 
-  const txs = await userWallet0.getTransactions()
-  expect((first(txs) as any).description).toBe(memo)
+  let firstTxs
+
+  const txs: Record<string, string>[] = await userWallet0.getTransactions()
+  firstTxs = first(txs)
+  if (!firstTxs) {
+    throw Error("No transactions found")
+  }
+  expect(firstTxs.description).toBe(memo)
 
   // receiver should not know memo from sender
   const txsUser3 = await userWallet3.getTransactions()
-  expect((first(txsUser3) as any).description).not.toBe(memo)
+  firstTxs = first(txsUser3)
+  if (!firstTxs) {
+    throw Error("No transactions found")
```

Comment:
Same issue as above. Is this the idiomatic way to fail a test? Can't we guarantee the return type to be not nil?

Reply:
Is it never null? That would be cleaner.